src/main/resources/hudson/plugins/ec2/EC2Cloud/provision.js

```diff
@@ -0,0 +1,8 @@
+Behaviour.specify("[data-type='ec2-provision']", 'ec2-provision', -99, function(e) {
+  e.addEventListener("click", function (event) {
+    const form = document.querySelector("form[name='provision']");
```

Comment:
That query is problematic I think. The same query is used for the google-compute-engine-plugin and if both plugins are installed and configured, then it potentially chooses the wrong one. Ideally the query only select the form from the surrounding `td`

Reply:
you're right, also with just this plugin you can have multiple EC2 clouds. I'll fix that.